CONTRIBUTING.md

```diff
@@ -10,33 +10,73 @@ please read the [code of conduct](CODE_OF_CONDUCT.md).
 ```sh
 $ git clone https://github.com/netlify/netlify-cms
 $ cd netlify-cms
-$ yarn
+$ yarn start
```

Comment:
@erquhart Is there a reason we need to have contributors use `npm` here, or is that just a problem in the `package.json`?

Reply:
The yarn concurrency problem applies when using `lerna run` to run a named script in every package, so we mostly just do it in npm scripts.